web/admin/src/CategoryCell.js

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2018 DoubleDutch, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { Component } from 'react'
+import './App.css'
+
+export default class CategoryCell extends Component {
+  constructor() {
+    super()
+    this.state = {
+      isEditing : false
+  }
+}
+
+  letEdit = () => {
```

Comment:
It looks like this function is doing something that should be called `toggleEdit`